bin/elixir

```diff
@@ -210,7 +219,8 @@ set -- "$ERTS_BIN$ERL_EXEC" -pa "$SCRIPT_PATH"/../lib/*/ebin $ELIXIR_ERL_OPTIONS
 if [ -n "$RUN_ERL_PIPE" ]; then
   ESCAPED=""
   for PART in "$@"; do
-    ESCAPED=""$ESCAPED" $(echo "$PART" | sed 's/[^a-zA-Z0-9_\-\/]/\\&/g')"
+    PART_ESCAPED="$(echo "$PART" | sed 's/[^a-zA-Z0-9_\-\/]/\\&/g')"
+    ESCAPED="$(printf "%s %s" "$ESCAPED" "$PART_ESCAPED")"
```

Comment:
Can you please explain the difference?

Reply:
The linter complains about the quotes at the beggining. I would prefer if we could stick to it.

Reply:
I don't think we need the double quotes in $ESCAPED. Would this be enough: `ESCAPED="$ESCAPED $(echo "$PART" | sed 's/[^a-zA-Z0-9_\-\/]/\\&/g')"`?

Reply:
I think so.
I have pushed this suggestion, but tests are failing locally (most likely unrelated to this change)

Reply:
@eksperimental it is unrelated but it may have left some VMs hanging around, so you need to kill them before. I will fix them!